packages/pentaho/src/Card/CardMedia/CardMedia.tsx

```diff
@@ -0,0 +1,51 @@
+import { forwardRef } from "react";
+import {
+  ExtractNames,
+  HvBaseProps,
+  useDefaultProps,
+} from "@hitachivantara/uikit-react-core";
+
+import { staticClasses, useClasses } from "./CardMedia.styles";
+
+export { staticClasses as cardMediaClasses };
+
+export type HvCardMediaClasses = ExtractNames<typeof useClasses>;
+
+export interface HvCardMediaProps extends HvBaseProps {
```

Comment:
```suggestion
export interface HvCardMediaProps extends HvBaseProps<HTMLImageElement> {
```


We are also missing some descriptions below for props and the component ⬇️